lisa/tests/scheduler/eas_behaviour.py

```diff
@@ -493,11 +496,14 @@ class TwoBigThreeSmall(EASBehaviour):
 
     @classmethod
     def get_rtapp_profile(cls, plat_info):
+        # A duty cycle equivalent to the max capacity for all CPUs != big
```

Comment:
if that pattern is repeated, it needs to go into a method of EASBehaviour so all these tests share it.
Something like:
```
def get_big_duty_cycle(self)
     """
     Returns the smallest duty cycle percentage guaranteeing the task will be placed on the biggest CPUs of the system. This allows minimizing thermal issues while ensuring the expected placement.
     """
     ....
```